content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -522,24 +522,29 @@ This means that support is still available but will be removed in a future Kuber
 * [`rbd`](/docs/concepts/storage/volumes/#rbd) - Rados Block Device (RBD) volume
   (**deprecated** in v1.28)
 
-Older versions of Kubernetes also supported the following in-tree PersistentVolume types:
+The following in-tree PersistentVolume types are no longer available, but are supported through `CSIMigration` and have a corresponding CSI driver implemented.
 
 * [`awsElasticBlockStore`](/docs/concepts/storage/volumes/#awselasticblockstore) - AWS Elastic Block Store (EBS)
-  (**not available** in v1.27)
+  (**not available** starting with v1.27, see [AWS EBS](https://github.com/kubernetes-sigs/aws-ebs-csi-driver) CSI driver)
 * [`azureDisk`](/docs/concepts/storage/volumes/#azuredisk) - Azure Disk
-  (**not available** in v1.27)
+  (**not available** starting with v1.27, see [Azure Disk](https://github.com/kubernetes-sigs/azuredisk-csi-driver) CSI driver)
 * [`cinder`](/docs/concepts/storage/volumes/#cinder) - Cinder (OpenStack block storage)
-  (**not available** in v1.26)
+  (**not available** starting with v1.26, see [OpenStack Cinder](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/cinder-csi-plugin/using-cinder-csi-plugin.md) CSI driver)
+* [`gcePersistentDisk`](/docs/concepts/storage/volumes/#gcepersistentdisk) - GCE Persistent Disk
+  (**not available** starting with v1.28, see [GCE Persistent Disk](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver) CSI driver)
+
+Older versions of Kubernetes also supported the following in-tree PersistentVolume types:
+
 * `photonPersistentDisk` - Photon controller persistent disk.
-  (**not available** starting v1.15)
+  (**not available** starting with v1.15)
```

Comment:
We could rewrite this and comment that there's no CSI driver for Photon available from Kubernetes.

Reply:
There is no CSI driver from Kubernetes for all the types in this section, not only Photon, right?
Should we add a statement such as `These volume types do not have a CSIMigration plan and are discontinued in newer Kubernetes releases.` ?

Reply:
In the following page, we can find CSI drivers for Quobyte and StorageOS. However, since this is updated by maintainers of individual CSI drivers, the info can be stale as well. It's probably better not to mention whether there's a CSI driver or not as it could also change.
https://kubernetes-csi.github.io/docs/drivers.html

Reply:
I think we should say when there are _in project_ drivers available. Readers would want to know.

Reply:
Thanks @xing-yang and @sftim. I understand it now. I have made an attempt to add this information in my latest commit.